hack/installers/install-kustomize-linux.sh

```diff
@@ -2,10 +2,16 @@
 set -eux -o pipefail
 
 # TODO we use v2 for generating manifests, v3 for production - we should always use v3
-KUSTOMIZE_VERSION=${KUSTOMIZE_VERSION:-3.1.0}
+KUSTOMIZE_VERSION=${KUSTOMIZE_VERSION:-3.2.1}
```

Comment:
note to self - we should get rid of this variable, we should only ever use a single version

Reply:
I can amend, and remove this - add leave it as:

```
KUSTOMIZE_VERSION=3.2.1
```